python/arcticdb/arctic.py

```diff
@@ -188,9 +209,21 @@ def get_library(self, library: str) -> Library:
         -------
         Library
         """
-        return self[library]
-
-    def create_library(self, name: str, library_options: Optional[LibraryOptions] = None) -> None:
+        try:
```

Comment:
Blow up if `create_if_missing` is Falsey but `library_options` are provided?

Reply:
Done